Zend/zend.c

```diff
@@ -119,6 +119,17 @@ static ZEND_INI_MH(OnUpdateErrorReporting) /* {{{ */
 }
 /* }}} */
 
+static ZEND_INI_MH(OnUpdateErrorBacktraceRecording)
+{
+	if (!new_value) {
+		EG(error_backtrace_recording) = E_FATAL_ERRORS;
+	} else {
+		EG(error_backtrace_recording) = atoi(ZSTR_VAL(new_value));
```

Comment:
Please use the ZEND_STRTOL macro instead of `atoi`

Reply:
will do! I copy-pasted this from the above `OnUpdateErrorReporting`, so I wasn't aware of that macro.

Reply:
_yah_ maybe update that INI handler too then :/